acquisitions/src/main/resources/thunderjet/mod-orders/features/independent-acquisitions-unit-for-ordering-and-receiving.feature

```diff
@@ -0,0 +1,645 @@
+# created for MODORDERS-982
+@parallel=false
+Feature: Independent acquisitions unit for ordering and receiving
+
+  Background:
+    * url baseUrl
+
+    * callonce loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * callonce loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json, text/plain'  }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json, text/plain'  }
+
+    * configure headers = headersUser
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+    * def orderId1 = callonce uuid3
+    * def orderId2 = callonce uuid4
+    * def orderId3 = callonce uuid5
+    * def orderId4 = callonce uuid19
+    * def poLineId1 = callonce uuid6
+    * def poLineId2 = callonce uuid7
+    * def poLineId3 = callonce uuid8
+    * def poLineId4 = callonce uuid20
+    * def pieceId3 = callonce uuid9
+    * def pieceId4 = callonce uuid21
+    * def acqUnitId1 = callonce uuid10
+    * def acqUnitId2 = callonce uuid11
+    * def acqUnitId3 = callonce uuid12
+    * def acqUnitMembershipId1 = callonce uuid13
+    * def acqUnitMembershipId2 = callonce uuid14
+    * def acqUnitMembershipId3 = callonce uuid15
+    * def acqUnitMembershipId4 = callonce uuid16
+    * def acqUnitMembershipId5 = callonce uuid17
+    * def locationId = callonce uuid18
+    * def titleId1 = callonce uuid22
+
+  Scenario: Create required records to scenarios, fund, budget and location
+    # 1. Create acq unit 'acqUnitId1', 'acqUnitId2' and 'acqUnitId3'
+    * configure headers = headersAdmin
+    Given path 'acquisitions-units/units'
+    And headers headersAdmin
+    And request
+    """
+    {
+      "id": '#(acqUnitId1)',
+      "protectUpdate": true,
+      "protectCreate": true,
+      "protectDelete": true,
+      "protectRead": true,
+      "name": "testAcqUnit1"
+    }
+    """
+    When method POST
+    Then status 201
+
+    Given path 'acquisitions-units/units'
+    And headers headersAdmin
+    And request
+    """
+    {
+      "id": '#(acqUnitId2)',
+      "protectUpdate": true,
+      "protectCreate": true,
+      "protectDelete": true,
+      "protectRead": true,
+      "name": "testAcqUnit2"
+    }
+    """
+    When method POST
+    Then status 201
+
+    Given path 'acquisitions-units/units'
+    And headers headersAdmin
+    And request
+    """
+    {
+      "id": '#(acqUnitId3)',
+      "protectUpdate": true,
+      "protectCreate": true,
+      "protectDelete": true,
+      "protectRead": true,
+      "name": "testAcqUnit3"
+    }
+    """
+    When method POST
+    Then status 201
+
+    Given path 'acquisitions-units/memberships'
+    And headers headersAdmin
+    And request
+    """
+      {
+        "id": '#(acqUnitMembershipId5)',
```

Comment:
it is not used in the test, do we need it?

Reply:
It is need for admin user